src/main/groovy/com/netflix/gradle/plugins/packaging/SystemPackagingTask.groovy

```diff
@@ -67,7 +67,7 @@ public abstract class SystemPackagingTask extends AbstractArchiveTask {
         mapping.map('release', { parentExten?.getRelease()?:getClassifier() })
         mapping.map('version', { parentExten?.getVersion()?:project.getVersion().toString() })
         mapping.map('epoch', { parentExten?.getEpoch()?:0 })
-        mapping.map('user', { parentExten?.getUser()?:getPackager() })
+        mapping.map('user', { parentExten?.getUser()?:'root' })
```

Comment:
Defaulting to root is much safer, then getPackager():

 1. All top directories is root-owner;
 1. Packager-owned files make unspecified behaviour: if packager user present in system - this user will file owner, otherwise file will be root-owned;
 1. Packager user (developer user) in most case unrelated to users created by package (system user).